deno.json

```diff
@@ -7,9 +7,9 @@
     "noUncheckedIndexedAccess": true
   },
   "importMap": "./import_map.json",
+  "unstable": ["webgpu", "unsafe-proto", "fs"],
```

Comment:
We'd like to apply `--unstable-unsafe-proto` only to specific task.

related https://github.com/denoland/std/pull/6342

Reply:
What's the harm in enabling `--unstable-unsafe-proto` in all tests? Genuine question. Also is this test supposed to pass with `deno task test`?

https://github.com/denoland/std/blob/2258bf2628a97a03dece8d0235d910bfaf1f501d/assert/object_match_test.ts#L417-L435

Reply:
We should test without `--unstable-unsafe-proto` if we target Deno. We should test with `--unstable-unsafe-proto` if we target browser, node, bun, etc. Because we now target both groups of runtime (Deno) vs (Browser, Node, etc), it makes sense to run tests with and without `--unstable-unsafe-proto`.